src/main/java/frc/robot/subsystems/Balancer.java

```diff
@@ -0,0 +1,84 @@
+package frc.robot.subsystems;
+import com.kauailabs.navx.frc.AHRS;
+import edu.wpi.first.wpilibj.SPI;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Balancer extends SubsystemBase {
```

Comment:
i think it's more inline with the intent of the command based drivestructure to do this using a Command. Is there a way to cleanly implement this using a command? 

that way the default command will also be overriden by the balancecommand